### ISSUE TITLE
Test BLEU Score 

### ISSUE THREAD

**Issue by MukundKhandelwal** [OpenNMT/OpenNMT#562]
@guillaumekln 
Hi,
I want to know if there is an option to get the BLEU score on test data set. For example, the training step (shown below) evaluates the BLEU score on the evaluation data set. 
**onmt-main train_and_eval --model_type Transformer --config data.yml --auto_config --num_gpus 4**
However, I need to check the accuracy on my test data set as well. Could you please provide some insight on how I can check that.
Thanks.

**Comment by guillaumekln** [comment]
Looks like you are using the TensorFlow version. Please open this issue in the appropriate repository or on the forum.
